Gopkg.lock

```diff
@@ -113,6 +113,14 @@
   revision = "85d198d05a92d31823b852b4a5928114912e8949"
   version = "v2.9.0"
 
+[[projects]]
+  digest = "1:f1f2bd73c025d24c3b93abf6364bccb802cf2fdedaa44360804c67800e8fab8d"
```

Comment:
Why do we need this?

Reply:
Really not sure, i just ran dep ensure and it seems to have updated this.must have been a transitive dependency